wiki/osu!stream/de.md

```diff
@@ -0,0 +1,375 @@
+| **perfekter Trefferwert** | (Anzahl der Misses + Anzahl der 50er + Anzahl der 100er + Anzahl der 300er) \* 4 |
+
+Mit anderen Worten, jeder Miss ist 0%, jeder 50er ist 25%, jeder 100er ist 50% und jeder 300er ist 100% Genauigkeit wert.
+
+### Punkte
+
+Die Punkte, die von jedem Hit-Circle und jedem Ende eines Sliders vergeben werden, wird mit folgender Formel berechnet:
+
+`Punkte = Trefferwert + Trefferwert * (Combomultiplizierer * fester Multiplizierer) / 25`
+
+| Begriff | Bedeutung |
+| :-: | :-- |
+| **Trefferwert** | Die Bewertung des Hit-Circles (50, 100 oder 300). |
+| **Combomultiplizierer** | (Combo vor dem Treffer - 1) oder 0; abhängig davon, was höher ist. |
+| **Fester Multiplizierer** | Der Multiplizierer der den Punktestand bei 600.000 fixiert. |
+
+Zusätzlich gibt jeder Sliderstart, -ende und -repeat tick 30 Punkte und jede volle Umdrehung eines Spinners gibt 100 Punkte.
+Es gibt einen weiteren Bonus von 10 Punkten, abhängig von den Umdrehungen pro Minute nachdem die Spinnermessbar gefüllt ist.
+
+### Bewertung des Hit-Circles
+
+**Hit-Circles**
+
+- Eine 300, 100 oder 50 wird bei einem normalen Hit-Circle verliehen, abhängig von der Treffgenauigkeit.
+- Ein Miss wird vergeben, wenn ein Hit-Circle zu früh oder überhaupt nicht getroffen wurde.
+  - Wenn das Objekt viel viel zu früh getroffen wird, wackelt es stattdessen. Nichts wird ansosnten passieren, außer die Visualisierung, dass das Objekt zu früh getroffen wurde.
+
+**Hold-Circle**
+
+- Die Bewertung des Hold-Circles hängt ab von dem initialen Treffer und die Haltezeit, bis der Kreis explodiert.
+  - 300 bei perfekter Ausführung.
+  - 100 für einen leicht inkorrekten initialen Treffer und den Kreis richtig halten.
+  - 50 für eine leichte Berührung.
+  - 30 für jede Explosion.
+  - Der Multiplizierer wird zurückgesetzt, wenn die Kreise nicht korrekt gehalten wurden. Ein Miss gibt es, wenn die Kreise überhaupt nicht gehalten wurden.
+
+**Slider**
+
+- **Die Bewertung von Slidern hängt ab von der Genauigkeit des initialen Treffers**.
+- Slider bestehen aus Sliderticks, die den Start, das Ende und die repeat points des Sliders einbeziehen.
+  - Eine 300, wenn man alle Sliderticks erreicht und einen akkuraten initialen Treffer hat.
+  - Eine 100, wenn mindestens die Hälfte des Sliders mit einem guten initialen Treffer getroffen wurde.
+  - Eine 50, wenn mindestens ein Slidertick oder der initiale Treffer getroffen wurde.
+- **Einen Slider viel zu früh zu tippen** wird keinen Miss erzeugen, aber **wird den Punktemultiplizierer auf 0 zurücksetzen**.
+- Ein Miss, wenn der Slider überhaupt nicht gehalten wurde.
+
+**Spinner**
+
+- Für einen Spinner werden 300er, 100er oder 50er vergeben, abhängig von der Anzahl der gemachten Spins verglichen mit der Länge des Spinners.
+- Ein Miss, wenn die Spinnermessbar, die auf beiden Seiten des Spinners zu sehen ist, nicht gefüllt wurde.
```

Comment:
```suggestion
- Ein Miss, wenn die Spinnermessleiste, die auf beiden Seiten des Spinners zu sehen ist, nicht gefüllt wurde.
```

Might be  a personal thing but the first thing I think of when reading this is measurable spinners.